src/test/java/teammates/test/cases/testdriver/HtmlHelperTest.java

```diff
@@ -135,4 +135,20 @@ public void testConvertToStandardHtmlPart() throws Exception {
         
         //other cases are tested in testComparison
     }
+    
+    @Test
+    public void testReplacement() throws IOException {
```

Comment:
Either option doesn't sound good; whatever I've written in the Gists before (for reference, [this](https://gist.github.com/wkurniawan07/e1fb00be615bb34ca01f) and [this](https://gist.github.com/wkurniawan07/d5d7dc842fdb922bbcb9)) are quite detailed and technical, and would be against the "minimal yet sufficient documentation" principle. A technical report might be a better alternative.